src/sentry/spans/consumers/process/factory.py

```diff
@@ -94,7 +94,10 @@ def _build_snuba_span(relay_span: Mapping[str, Any]) -> MutableMapping[str, Any]
         sentry_tags["status"] = status
 
     if "status_code" in sentry_tags:
-        sentry_tags["status_code"] = int(sentry_tags["status_code"])
+        try:
+            sentry_tags["status_code"] = int(sentry_tags["status_code"])
```

Comment:
This is fine for now. But we could potentially send default values downstream if desired. In Snuba consumers we always assume that a value could be incorrect and in those cases populate a default value.